### ISSUE TITLE
make bot skip death mage in baal

### ISSUE THREAD

**Issue by Parme89** [kolton/d2bot-with-kolbot#606]
using a hammerdin and a merc, issue comes when bot always hunt the mages and wont let merc kill the unravels, it always teles around to the skeletons , takes years to get past them, is there a easy way of just tell the bot to skip those guys ?

**Comment by DarkHorseDre** [comment]
I think there may be some coding to make hammerdin kill skellies so merc can focus on immune but not sure: someone else can confirm.

What do you have 	`Config.BossPriority` set as?

**Comment by Parme89** [comment]
@DarkHorseDre as for now its set for false, i know coding but only lua . so the coding part can be a problem ^^

**Comment by DarkHorseDre** [comment]
ok so if you wanna skip the skellies, try setting boss priority to true, however hammer wont kill the magic immunes unless you have an alternative skill?  Smite works.  

I cant remember if its default, but hammerdin should try holy bolt. I read a post where someone was using holy bolt, and another re: smit but I cant remember where..  Smite should have 1 point already and with oskills is good - try that as attack against immunes (with concentration aura) - merc will attack too as pala shouldn't be teleporting i think.

**Comment by Parme89** [comment]
@DarkHorseDre thanks ! ill for sure try that . ill come back with an update soon

**Comment by master1g** [comment]
Holy Bolt should be used as secondary against immunes.  It is # 101

Your hammerdin will be able to kill the second wave with this, but it's not efficient.  When I run baals I run at least one other character with me, which you might consider.  With cdkeys being cheap from certain sites you def should start doing this as running baals with only a hammerdin is not efficient and it won't work in high player games if you're the main damage dealer.  You make something as cheap as a blizzard sorc and fly through the second wave like it's nothing.  Hope this helps.

**Comment by Parme89** [comment]
@master1g as a hammerdin has smite already skilled it works like a charm , i did like @DarkHorseDre said and now i have no problems doing baal while he focus smiting the unravels the merc can kill them easily , thanks both of u for great suggestions !

**Comment by Parme89** [comment]
@DarkHorseDre Yeah thats correct, just started so didnt have a char to follow , starting a lightsoso now that will assist ;)
